src/scripts/main.js

```diff
@@ -1,3 +1,7 @@
 'use strict';
 
-// write code here
+const body = document.querySelector('body .wall .spider');
```

Comment:
Why body if it is a spider? Also you can use only `.spider` selector, no need to to list all its parents.